src/Components/Button.css

```diff
@@ -0,0 +1,27 @@
+.button {
+  width: 100%;
+  padding: 15px;
+  font-size: 18px;
+  background-color: #fff;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+}
+
```

Comment:
- [ ] You have done good work 🥇, however, let's make our  React component looks like the design provided as it is [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A) 😃.  Kindly fix it 👍 

> | | Original design | | Your design | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-08-08 181021](https://github.com/lerfast/math-magicians/assets/57408419/bd1eef40-3f0d-4226-85a7-58d7fad5e9ef)| |![Screenshot 2023-08-08 181105](https://github.com/lerfast/math-magicians/assets/57408419/da2dee13-fd24-4d9f-a7c8-70b286234fa0)| |